apps/semver-yeasy/semver-yeasy.sh

```
@@ -241,29 +242,29 @@
-        # ${GITVERSION_EXEC_PATH} $(pwd) /nonormalize /config "${svc}/.gitversion.yml"
-        gitversion_calc=$(${GITVERSION_EXEC_PATH} $(pwd) /nonormalize /config "${svc}/.gitversion.yml")
+            echo "calculation for ${svc}"
+            CONFIG_FILE="${!CONFIG_FILE_VAR}"
+            CONFIG_FILE=$(echo "${CONFIG_FILE}" | sed "s|\$svc|$svc|")
 
-        GITVERSION_TAG_PROPERTY_NAME="GITVERSION_TAG_PROPERTY_$(echo "${DIFF_SOURCE}" | sed 's|/.*$||' | tr '[[:lower:]]' '[[:upper:]]')"
-        GITVERSION_TAG_PROPERTY=${!GITVERSION_TAG_PROPERTY_NAME}
-        if [ "${GITVERSION_TAG_PROPERTY}" == "" ]; then
-            GITVERSION_TAG_PROPERTY=${GITVERSION_TAG_PROPERTY_DEFAULT}
-        fi
+            gitversion_calc=$(${GITVERSION_EXEC_PATH} $(pwd) /nonormalize /config "${CONFIG_FILE}")
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)